### PR TITLE
[terra-application-links] Remove Terra.it helpers from wdio 

### PR DESCRIPTION
**Pull request by saket2403**
### Summary
Remove Terra.it helpers from wdio  as they are deprecated.

### Deployment Link

https://terra-framework-deployed-pr-#.herokuapp.com/

### Additional Details

Thank you for contributing to Terra.
@cerner/terra

### CODE REVIEW

File: packages/terra-application-links/tests/wdio/application-tabs-spec.js [hunk 1]
```diff
@@ -79,29 +66,23 @@ Terra.describeViewports('ApplicationTabs', ['small', 'large'], () => {
...
-      browser.click('[data-application-tab-menu-content] > *:first-child');
-      browser.pause(150);
-      browser.click('#test-tabs [data-application-tabs-more]');
-      browser.waitForVisible('[data-application-tab-menu-content]');
-      browser.moveToObject('[data-application-tab-menu-content] > *:first-child');
-    });
-
-    Terra.it.validatesElement({ selector: '[data-application-tab-menu-content]' });
+  it('Displays collapsed application tabs with icons', () => {
+    browser.url('/#/raw/tests/terra-application-links/application-links/application-tabs-with-icons');
```
Comment (emilyrohrbough) [inline packages/terra-application-links/tests/wdio/application-tabs-spec.js:77]:
```suggestion
    browser.url('/raw/tests/terra-application-links/application-links/application-tabs-with-icons');
```

File: packages/terra-application-links/tests/wdio/application-tabs-spec.js [hunk 2]
```diff
@@ -79,29 +66,23 @@ Terra.describeViewports('ApplicationTabs', ['small', 'large'], () => {
...
-  describe('Displays an application tabs with icons', () => {
-    beforeEach(() => {
-      browser.url('/#/raw/tests/terra-application-links/application-links/application-tabs-with-icons');
-      browser.waitForVisible('#test-tabs [data-application-tabs-more]');
-      browser.moveToObject('#test-tabs [data-application-tabs-more]');
-    });
-
-    Terra.it.validatesElement();
+  it('Displays an application tabs with icons', () => {
+    browser.url('/#/raw/tests/terra-application-links/application-links/application-tabs-with-icons');
```
Comment (emilyrohrbough) [inline packages/terra-application-links/tests/wdio/application-tabs-spec.js:70]:
```suggestion
    browser.url('/raw/tests/terra-application-links/application-links/application-tabs-with-icons');
```

File: packages/terra-application-links/tests/wdio/application-tabs-spec.js [hunk 3]
```diff
@@ -27,43 +24,33 @@ Terra.describeViewports('ApplicationTabs', ['small', 'large'], () => {
...
-    });
-
-    Terra.it.validatesElement();
+  it('Displays a application tabs end aligned', () => {
+    browser.url('/#/raw/tests/terra-application-links/application-links/application-tabs-end');
+    browser.pause(50);
+    Terra.validates.element('end aligned');
   });
 
   describe('Navigates away from the site', () => {
```
Comment (emilyrohrbough) [inline packages/terra-application-links/tests/wdio/application-tabs-spec.js:56]: Can we clean up the beforeEach hook here as well

File: packages/terra-application-links/tests/wdio/application-tabs-spec.js [hunk 4]
```diff
@@ -27,43 +24,33 @@ Terra.describeViewports('ApplicationTabs', ['small', 'large'], () => {
...
 
-  describe('Displays a application tabs end aligned', () => {
-    beforeEach(() => {
-      browser.url('/#/raw/tests/terra-application-links/application-links/application-tabs-end');
-      browser.pause(50);
-    });
-
-    Terra.it.validatesElement();
+  it('Displays a application tabs end aligned', () => {
+    browser.url('/#/raw/tests/terra-application-links/application-links/application-tabs-end');
```
Comment (emilyrohrbough) [inline packages/terra-application-links/tests/wdio/application-tabs-spec.js:51]:
```suggestion
    browser.url('/raw/tests/terra-application-links/application-links/application-tabs-end');
```

**Comment by saket2403** [comment]
Resolved all comments 6e4ab6b

**Review by emilyrohrbough — APPROVED** [review]

**Review by mjhenkes — APPROVED** [review]

**Pull request merged**